graph_pit/loss/optimized.py

```diff
@@ -29,6 +47,11 @@ class OptimizedGraphPITLoss(GraphPITBase):
         [1] Speeding up Permutation Invariant Training
     """
     assignment_solver: Union[Callable, str] = 'optimal_dynamic_programming'
+    reduction: str = 'sum'
+
+    def __post_init__(self):
+        super().__post_init__()
+        assert self.reduction in ('sum', 'mean')
```

Comment:
Can you add `, self reduction` for a better error message?

Reply:
Done